module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,17 @@
+(ns clojure.core.typed.test.template
+  (:require 	[clojure.core.typed :as t]
+                [clojure.template :as template] 
+		[clojure.xml :as xml]
+		[clojure.zip :as zip]
+		[clojure.test :refer :all]                
+		[clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest applyTemplate-test
+(is-tc-e (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))t/Any) 
+(is-tc-err (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)) t/Str))
+
+(deftest doTemplate-test
+
+	(is-tc-e (clojure.template/do-template [x y] (+ y x) 2 4 3 5) t/Any)
+	(is-tc-err (clojure.template/do-template [x y] (+ y x) 2 4 3 5) t/Str))
+
```

Comment:
Pleeease format this file correctly. Also please add more cases for each function. E.g. in both `apply-template` and `do-template` second argument can be anything. You can add some cases where second argoument is a map, symbol or something else.

Also when you write `is-tc-err` tests think about what errors might other users do when they use `apply-template` or `do-template` which can be caught by type checking.